inc/notificationevent.class.php

```diff
@@ -121,12 +121,12 @@ public static function raiseEvent($event, $item, $options = [], $label = '')
                                         PluginMreportingNotification::send(
                                             $template->getDataToSend(
                                                 $notificationtarget,
-                                                $tid,
+                                                (string)$tid,
```

Comment:
![image](https://github.com/user-attachments/assets/ee4c11fe-033c-461f-901f-077c2b569400)

To correct, you just need to run: `vendor/bin/php-cs-fixer fix .`